src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1082,6 +1119,15 @@ theorem sshiftRight_eq_of_msb_true {x : BitVec w} {s : Nat} (h : x.msb = true) :
         Nat.not_lt, decide_eq_true_eq]
       omega
 
+@[simp] theorem getElem_sshiftRight {x : BitVec w} {s i : Nat} (h : i < w) :
```

Comment:
Skeptical this should be `@[simp]`: generally if the RHS starts with an if we don't let simp proceed.